.github/workflows/publish.yaml

```
@@ -0,0 +1,55 @@
+name: CI/CD
+
+on:
+  push:
+    branches:
+      - "*"
+    tags:
+      - "*"
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

```suggestion
        python-version: [3.7, 3.8, 3.9, 3.10]
```
python3.6 dies in 2 weeks 3 days...